StockTrader/src/main/resources/schema.sql

```diff
@@ -75,13 +79,13 @@ BEGIN
         RETURN NULL;
       END IF;
 --       INSERT INTO account_transaction_log (action, funds, timestamp, username)
-      INSERT INTO account_transaction_log (action, funds, timestamp, username)
-      VALUES (action, funds, trunc(extract(epoch from now()) * 1000), NEW.user_id);
-      WITH temp (action,funds,timestamp,username) AS (values (action, funds, trunc(extract(epoch from now()) * 1000), NEW.user_id))
+      INSERT INTO account_transaction_log (action, funds, timestamp, username, server, "transactionNum")
+      VALUES (action, funds, trunc(extract(epoch from now()) * 1000), NEW.user_id, NEW.last_server, NEW.last_transaction_number);
```

Comment:
same here, whats with the quotes

Reply:
Same as above. To enforce capitalizing the column name. Postgres wants everything to be lower case.